go/teams/get_test.go

```diff
@@ -27,6 +27,28 @@ func TestTeamGet(t *testing.T) {
 	}
 }
 
+func TestGetFromID(t *testing.T) {
+	tc := libkb.SetupTest(t, "team", 1)
+	tc.Tp.UpgradePerUserKey = true
+	kbtest.CreateAndSignupFakeUser("team", tc.G)
+	defer tc.Cleanup()
+
+	name := createTeam(tc)
```

Comment:
I tried getting a team from the database by ID, but I don't think it has people in it yet so the API call failed when I tried to retrieve it. So I just made a new team and tried retrieving it by the ID it returned.